pipelines/rj_pic__disparos_email/email_sender.py

```
@@ -0,0 +1,200 @@
+        """
+        display_name = recipient_name or to_email
+
+        for attempt in range(1, MAX_RETRIES + 1):
+            try:
+                self._send_email_internal(to_email, subject, html_body, from_email)
+
+                # Sucesso
+                success_msg = f"E-mail enviado com sucesso para {display_name} ({to_email})"
+                logging.info(success_msg)
+                success_logger.info(f"{to_email} | {display_name}")
+                return True
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)